census/estimator/trainer/task.py

```diff
@@ -82,7 +82,7 @@ def run_experiment(hparams):
       '--train-files',
       help='GCS or local paths to training data',
       nargs='+',
-      required=True
+      default=['gs://cloud-samples-data/ml-engine/census/data/adult.data.csv']
```

Comment:
I submitted a new PR https://github.com/GoogleCloudPlatform/cloudml-samples/pull/276/commits/0a43f62e6ff3f805e25906f62698aa47c5a87347 which does't require the list parameter. I will approve for now.
If you can approve later that PR and then update the changes here will be good too.

Reply:
okay - will wait until your PR is merged and then update this.

Reply:
PR merged